features/post-generate.feature

```diff
@@ -41,11 +41,24 @@ Feature: Generate new WordPress posts
       """
       Success:
       """
-  Scenario: Generating post and output title
-    When I run `wp post generate --count=1 --post_title=Howdy!`
-    And I run `wp post list --field=post_title --posts_per_page=1`
+
+  Scenario: Generating post and outputting title and name
+    When I run `wp post generate --count=3 --post_title=Howdy!`
+    When I run `wp post list --format=count`
+    And save STDOUT as {POST_COUNT}
```

Comment:
This is not exactly how I meant using `{POST_COUNT}`. I assumed that you would get random IDs attached to the slugs and titles, so you'd use `{POST_COUNT}` to calculate the IDs that would be generated.

However, it looks like the empty site will always start with the same IDs either way, so that is not needed.

You can remove lines 47 & 48, and just hard-code the `--posts_per_page` instead.

Sorry for misleading you, I thought we would get more random IDs.

Reply:
No problem @schlessera I have hard-coded the number of posts. please check.